### ISSUE TITLE
Pause/mute playback when window loses focus

### ISSUE THREAD

**Issue by gingerbeardman** [benwiggy/APPlayMIDI#2]
When opening/auditioning multiple MIDI files it's annoying to have to manually pause one and switch to another. 

It would be nice if audio paused or muted when window lost focus.

**Comment by benwiggy** [comment]
Yes, though some people may want to leave it playing in the background. Perhaps make sure only one window is playing at any one time?

**Comment by matronator** [comment]
I second this. Normal way quicklook works is that it plays only one item at a time in the same Finder window, but can play multiple items at once if they're each in separate window.

Because if you have like a lot of MIDI samples/loops you want to preview, instead of just pressing the down button, you have to stop the current one and then go to the next one.

You could still let it play in the background, but just not in the same window.

**Comment by maxoakland** [comment]
I have this issue too
